cmd/minikube/cmd/root.go

```diff
@@ -101,7 +101,8 @@ func Execute() {
 
 	_, callingCmd := filepath.Split(os.Args[0])
 
-	if callingCmd == "kubectl" {
+	isExe := callingCmd == "kubectl.exe" && runtime.GOOS == "windows"
```

Comment:
could it also be called as kubectl on windows ? (without user typing .exe) 
so maybe we should call it isExe in case of `kubectl` && runtime.GOOS == "windows ?

Reply:
The line below is `if callingCmd == "kubectl" || isExe`
So if they're calling `kubectl` on Windows it will be caught by `callingCmd == "kubectl"`